test/metabase/pulse/render/body_test.clj

```diff
@@ -231,7 +231,7 @@
                                 :coercion_strategy :Coercion/ISO8601->DateTime}))
 
 (deftest cols-with-semantic-types
-  (is (= [{:bar-width nil, :row [(number "1") (number "34.10") "Apr 1, 2014" "Stout Burgers & Beers"]}
+  (is (= [{:bar-width nil, :row [(number "1") (number "34.1") "Apr 1, 2014" "Stout Burgers & Beers"]}
```

Comment:
I'm confused by this change. I think these are the values 34.0996, 34.0406, 34.0474 from `test-data`. I don't see why the first one now is rounded to one decimal place and the other two go to two decimal places.

Reply:
This is how the frontend displays the value 34.0996 actually. It first rounds the least-significant-digit, since there are 3 sig-digs (by default, it will round to 2). This gives the value 34.100, which then has trailing zeroes stripped. 

This only happens because the number of decimal places is not explicitly set.

You can verify it in the UI if you make a new SQL question and do:
`select 0.0996 A, 34.0996 B`

<img width="173" alt="image" src="https://user-images.githubusercontent.com/21064735/196561209-cdec3094-ec98-4e22-8e3b-48cbcb5d7ab2.png">